framework/auth/core.py

```diff
@@ -1122,6 +1122,11 @@ def get_daily_digest_log_ids(self):
         )
         return self.get_recent_log_ids(since=midnight)
 
+    def files_checked_out(self):
+        # done here to prevent Import error due to circular import.
+        from website.files.models.osfstorage import FileNode
+        return list(FileNode.find(Q('checkout', 'eq', self)))
```

Comment:
I don't think you need to cast this into a list since you can still iterate over the queryset.